src/Cut/Lib.hs

```diff
@@ -41,10 +41,12 @@ main = do
   liftIO $ print options
 
   parsed <- detect options
-  case options ^. work_dir of
-    Nothing ->
-      withTempDirectory "/tmp" "streamedit" $ liftIO . runEdit options parsed
-    Just x -> liftIO $ runEdit options parsed x
+  case parsed of
+    [] -> liftIO $ putStr "\n\nNo silence detected\n\n"
```

Comment:
Yeah, this is the idea!

I'd make this message be: No silence in input video detected. There is nothing to be cut so exiting.

Reply:
We want to make our error messages precise.

Reply:
Message changed